### DISCUSSION TITLE
Please change e107 so that it is possible to have another admin theme..

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5370] [General]
I tried to use the standard bs5 as admin theme in addition to the standard bs3 theme but that gives very strange situations, in the theme config there is the option to set bs5 as admin but it doesn't work, so now I have to keep the name bootstrap3 while everything in it works with bootstrap 5, including the front end because otherwise all kinds of other problems arise...

**Comment by Jimmi08** [comment]
For now, 
go to admin/auth.php and look for this code:

```
if($adminTheme !== 'bootstrap3'/* && $adminTheme !== 'bootstrap5'*/)
{
	$core->update('admintheme','bootstrap3');
	$core->update('adminstyle','infopanel');
	$core->update('admincss','css/modern-light.css');
	$core->set('e_jslib_core',array('prototype' => 'none', 'jquery'=> 'auto'));
	$core->save();	
	e107::getRedirect()->redirect(e_SELF);		
}
```

uncomment bootstrap5 part. For theme manager both are allowed:
private $approvedAdminThemes = array('bootstrap3', 'bootstrap5');

but that part above is hardcoded to bootstrap3.

**Reply by Alex-e107nl** [reply]
@Jimmi08 Thank you!, because it is simply not possible to use a different name and thus have a 2nd admin theme, nobody burns their fingers to try it, for now it would be nice if the standard bootstrap5 theme could also be set up as admin so that you can switch, hence the request for the override for the core template admin_template.php

**Reply by Jimmi08** [reply]
You put this in discussion, you need to create a Feature request for them to consider this change. For testing theme, it is enough. 
I would like to know why it is commented - maybe they run into unknown issue for us...

**Comment by Alex-e107nl** [comment]
You are right of course but I would like to have a discussion first, if there is a negative response I don't have to make any further effort and I will finish the theme and use it by overwriting the files that are so outdated, really, btn-mini or even btn-xs? no nav-link / nav-item download-item and everywhere different ways to build a link or a menu, everywhere inline css, I am so happy that css !important exists! I will just wait for the response and continue working on the theme, almost every part needs to be checked, adjusted and tested, 

I will still be busy with it for a while... A big advantage for me is that with the dark light buttons everyone is always satisfied, I like to work with a light version but I also know users who swear by a dark version and now everything is in 1... I even had to make an adjustment in the theme_handler.php because the config page for the theme does not produce real tabs..
